src/gmastertrack.cxx

```diff
@@ -170,11 +170,16 @@ static void gmastertrack_button_callback(Fl_Widget *w, void *data)
 
 	} else if ( strcmp( w->label(), "Tap" ) == 0 ) {
 		if ( Fl::event_button() == FL_RIGHT_MOUSE ) {
-			const char* answer = fl_input("Enter BPM value: ");
+			string question = "Enter BPM value (between " + to_string(MIN_TEMPO) + " and " 
+								+ to_string(MAX_TEMPO) + "):";
```

Comment:
There's a "trailing whitespace" here, see the # symbol here:    " and "#

Reply:
I did this on purpose because I hate vertical scrolling with my touchpad. :D Should be fixed.